recipe/meta.yaml

```diff
@@ -12,17 +12,19 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win]
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
+    - {{ compiler('c') }}
```

Comment:
The cython runtime dependency should handle this, no?

Reply:
It may be possible to remove cython, being that cyipopt depends on it. Should investigate.

Reply:
No that is part of the `conda-forge` setup and a python dependency won't trigger the compiler installation.

Reply:
This package doesn't need a compiler at build time. It only needs it at run time. It uses cython like a JIT compiler.

Reply:
Makes sense, the migrator does not handle that.